src/Future.js

```diff
@@ -33,7 +33,7 @@ Future.prototype.ap = function(m) {
 
   return new Future(function(rej, res) {
     var applyFn, val;
-    var doReject = R.once(rej);
+    var doReject = rej != null ? R.once(rej) : function() {};
```

Comment:
`null` being passed to `fork`. I fully agree with your preference. I was just hesitant to introduce a potentially breaking change (though admittedly, nothing officially documents (afaik) that `fork` can accept `null` as a noop). I'll update.